.github/workflows/integration.yml

```
@@ -1,20 +1,76 @@
+      - name: Setup application.properties
+        run: |
+          mkdir -p ./src/main/resources
+          echo "${{ secrets.APPLICATION_PROPERTIES }}" > ./src/main/resources/application.properties
+          echo "${{ secrets.PRODUCTION_PROPERTIES }}" > ./src/main/resources/prod.properties
+
+      - name: Generate Swagger JSON
+        run: |
+          ./gradlew generate generateOpenApi
+
+      - name: Save swagger json into Actions artifact
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.4)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)